traveler-swift-core/TravelerKit/AuthPath.swift

```diff
@@ -117,8 +117,17 @@ enum AuthPath {
             }
         case .cancellationQuote(let order):
             urlComponents.path = "/v1/order/\(order.id)/cancellation"
-        case .cancelOrder(let quote):
-            urlComponents.path = "/v1/order/\(quote.order.id)/cancellation/\(quote.id)"
+        case .cancelOrder(let request):
+            urlComponents.path = "/v1/order/\(request.quote.order.id)/cancellation/\(request.quote.id)"
+            urlComponents.queryItems = [
```

Comment:
This cannot be in the queryparams....

only GET requests use the query params....Things like this should be brought up with the backend dev working on it

Reply:
Will hold PR until resolved

Reply:
It's resolved in backend.